sstables/sstables.cc

```diff
@@ -961,7 +961,7 @@ void sstable::filesystem_storage::open(sstable& sst, const io_priority_class& pc
         // the generation of a sstable that exists.
         w.close();
         remove_file(file_path).get();
-        throw std::runtime_error(format("SSTable write failed due to existence of TOC file for generation {:d} of {}.{}", sst._generation, sst._schema->ks_name(), sst._schema->cf_name()));
+        throw std::runtime_error(format("SSTable write failed due to existence of TOC file for generation {:d} of {}.{}", sst._generation.value(), sst._schema->ks_name(), sst._schema->cf_name()));
```

Comment:
This work as a stop-gap solution but the real solution is to make the generation formattable. No need to do it now, we can defer that to whoever is working on making the generation an UUID.

Reply:
> This work as a stop-gap solution but the real solution is to make the generation formattable.

Right

> No need to do it now, we can defer that to whoever is working on making the generation an UUID.

@jul-stas